components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounterMenu.kt

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.ui.tabcounter
+
+import android.content.Context
+import androidx.core.content.ContextCompat.getColor
+import mozilla.components.browser.menu2.BrowserMenuController
+import mozilla.components.concept.menu.MenuController
+import mozilla.components.concept.menu.candidate.DrawableMenuIcon
+import mozilla.components.concept.menu.candidate.TextMenuCandidate
+import mozilla.components.concept.menu.candidate.TextStyle
+
+open class TabCounterMenu(
+    context: Context,
+    private val onItemTapped: (Item) -> Unit
+) {
```

Comment:
Should we force the user to customize this? How can we make it both generic enough that the "plain" version will work, but that users can also customize it?